exasol_error_reporting_python/placeholder_handler.py

```diff
@@ -0,0 +1,42 @@
+import re
+from typing import Any, Iterable
+
+from exasol_error_reporting_python.placeholder import Placeholder
+
+
+class PlaceholderHandler:
+    PLACEHOLDER_PATTERN = "{{([^}]*)}}"
+    PLACEHOLDER_NONE = "<null>"
+
+    @classmethod
+    def get_placeholders(cls, text: str) -> Iterable[Placeholder]:
+        for match in re.finditer(cls.PLACEHOLDER_PATTERN, text):
+            yield Placeholder(
+                key=match.group(0),
+                text=match.group(1))
+
+    @classmethod
+    def replace_placeholder(cls, text: str, parameter_dict: dict) -> str:
+        for placeholder in cls.get_placeholders(text):
+            replacement = cls.get_parameter_from_dict(
+                placeholder, parameter_dict)
+            text = text.replace(placeholder.key, replacement)
+        return text
+
+    @classmethod
+    def get_parameter_from_dict(cls, placeholder: Placeholder,
+                                parameter_dict: dict) -> Any:
+
+        parameter = parameter_dict.get(placeholder.name, None)
+
+        if not parameter:
```

Comment:
do you want to test for not None, then please use `is not None`, in general, I prefer more explicit comparison, in contrast to python, for example `if len(some_list)==0` instead `if some_list`